### DISCUSSION TITLE
How to store custom cookie in justpy application .

### DISCUSSION THREAD

**Discussion by Rushikesh210** [elimintz/justpy#691] [Q&A]
How to store custom cookie in justpy application .

**Comment by WolfgangFahl** [comment]
I am continuously frustrated that questions like this are opened as issues and the [if you have questions-suggestions-or would like to get in touch](https://justpy.io/#if-you-have-questions-suggestions-or-would-like-to-get-in-touch) is completly ignored.

**Reply by Rushikesh210** [reply]
Sorry for the inconvenience, from next time I am put any questions on issues.
